lib/util/range.ts

```diff
@@ -0,0 +1,13 @@
+export function* range(
+  start: number,
+  end: number
+): Generator<number, void, void> {
+  if (start > end) {
+    return;
+  }
+  yield start;
+  if (start === end) {
+    return;
+  }
+  yield* range(start + 1, end);
```

Comment:
It's low risk, but this is a recursive function to generate a list of numbers. This seems a little overkill, and has the potential to exceed the max stack size for large lists. A for loop seems easier and safer?

Reply:
```suggestion
  for (let i = start; i < end; i++) {
    yield i;
  }
```

Reply:
Good suggestion, thanks 👍